ocspresponder.go

```diff
@@ -15,7 +18,63 @@ type CertDbSource struct {
 	Accessor certdb.Accessor
 }
 
-func NewSource(dbAccessor certdb.Accessor) cfocsp.Source {
+const interval = 96 * time.Hour
+
+func NewSource(dbAccessor certdb.Accessor, caFile string, respFile string, respKey string) cfocsp.Source {
+
+	go func() { // heavy join
+		signer, err := cfocsp.NewSignerFromFile(caFile, respFile, respKey, interval)
```

Comment:
Accept a signer as a parameter, you shouldn't need to create it yourself and you don't know how your caller might want to set it up